actions/csm-k8s-api-checker/action.yaml

```diff
@@ -103,16 +107,25 @@ runs:
           ${{ inputs.prerequisite }}
 
           dir=${{ inputs.chart-directory }}
+
+          # The following converts "k8s=v1.21.0,cert-manager=v0.14.1" to "--target-versions k8s=v1.21.0 --target-versions cert-manager=v0.14.1 "
+          component_versions=$(echo ${{ inputs.PLUTO_COMPONENT_VERSIONS }} | awk -v RS=',' '{if($1!=""){printf "--target-versions %s ", $1}}' )
+
           complain=false
 
-          if [ "$(pluto detect-files --ignore-deprecations --ignore-removals --directory $dir 2>&1)" != "There were no resources found with known deprecated apiVersions." ]; then
+          if ! pluto detect-files --directory $dir $component_versions; then
             complain=true
-            pluto detect-files --output custom --columns DEPRECATED,REMOVED,NAME,KIND,VERSION --ignore-deprecations --ignore-removals --directory $dir > lhs
-            pluto detect-files --output custom --columns REPLACEMENT --ignore-deprecations --ignore-removals --directory $dir > rhs
-            paste lhs rhs | sed '/^[[:space:]]*$/d' > plutooutput
+            pluto detect-files --directory $dir $component_versions --ignore-deprecations --ignore-removals --output custom --columns DEPRECATED,REMOVED,NAME,KIND,VERSION,COMPONENT,'DEPRECATED IN','REMOVED IN' > lhs
```

Comment:
Looking at this I wonder if we don't just nuke DEPRECATED and REMOVED in this pr too, if we have deprecated/removed in instead, the true/false data is just redundant and crowding out column space needlessly.

Reply:
Could the columns be moved to the details (rhs) where the REPLACEMENT column is?
The removed might be useful because `pluto list-versions` shows things that will be removed in the future. I assume the pluto tool knows the current version of k8s. The current version is not explicit in the output.

Reply:
Sorta, the current version if not specified is "whatever the pluto version considers the latest at the time you run it". Aka I think thats 1.26 for now but a month or two ago was 1.25.

All the lhs/rhs stuff is is just me working around the pluto tool not honoring the column order we give to put REPLACEMENT at the end of the table as there might be removed apis with no replacement and that messes up markdown formatting and shifts columns over by one. It is a bit of a hack to make markdown tables easier...ish to make in shell if that possibly empty column is at the end.

I just don't think it makes sense to have both columns one saying yeah there are removed things with another that says yeah its replaced with this is all.